src/Components/Slides/SlidesForm.js

```diff
@@ -1,32 +1,107 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
+import axios from "axios";
+import React, { useState } from "react";
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import "../FormStyles.css";
+import Swal from 'sweetalert2'
 
-const SlidesForm = () => {
-    const [initialValues, setInitialValues] = useState({
-        name: '',
-        description: ''
-    });
-
-    const handleChange = (e) => {
-        if(e.target.name === 'name'){
-            setInitialValues({...initialValues, name: e.target.value})
-        } if(e.target.name === 'description'){
-            setInitialValues({...initialValues, description: e.target.value})
+const SlidesForm = ({slides}) => {
+  const [initialValues, setInitialValues] = useState({
+    name: slides?.name ?? '',
+    description:slides?.description ?? '',
+    image:slides?.image ?? '',
+    order:slides?.order ?? '',
+  });
+  const hasSlides = !!slides
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  const handleData = () => {
+    if(!hasSlides){
+    axios({
+      method: "POST",
+      url: "http://ongapi.alkemy.org/api/slides",
+      data: {
+        name: initialValues.name,
+        description: initialValues.description,
+        image: initialValues.image,
+      },
+    })
+      .then((res) => {
+        if (initialValues.name.length>4) {
+          Swal.fire(
+            'Slide Creado!'
+          )
```

Comment:
this Swal.fire is frontend logic so on the handleData method you should call something like this:
```
const showSuccessMessage(message = "success") => Swal.fire(message);
const showErrorMessage(message, confirmButtonText) => {
    swal code here
};
...code here

slideService.createOrUpdate(slide)
    .then(()=> showSuccessMessage())
    .catch(error => showErrorMessage(error.errorMessage, 'Ok, voy a editarlo!'));
```

Reply:
i was not able to change all  of this, but i tried, maybe you can help me with the succes alert